docs/release-notes.txt

```diff
@@ -4,6 +4,112 @@
+   /some/path`` will have two effects: first, ``/some/path`` will be bind-mounted into the container
+   (still as ``/some/path``); second, interactive jobs (notebooks, shells, and commands) will run in
+   the provided working directory by default. Note that containers run as the root user by default,
+   so you may want to :ref:`configure your user <run-as-user>` with ``det user`` such that
+   interactive jobs run as your regular user.
+
+-  Support configuring the working directory using the ``work_dir`` configuration field for
+   commands, shells, and notebooks. You can also optionally set it in the
+   ``task_container_defaults.work_dir`` field of the master configuration. The value set in the
+   master configuration will be ignored when a context directory is submitted.
+
+-  WebUI: Allow experiment owners to delete their own experiments, singly or in batches.
+
+   .. image:: https://user-images.githubusercontent.com/220971/134048799-cd663a75-cb24-4f44-9a8a-c2ff23222cef.png
+
+   .. image:: https://user-images.githubusercontent.com/220971/133659677-aea0d1bc-95ce-4652-8218-92b97d114358.png
+
+-  WebUI: Display the latest log entry available for a trial at the bottom of the trial's page. This
+   works for both single-trial experiments and trials within a multi-trial experiment.
+
+   .. image:: https://user-images.githubusercontent.com/220971/131391658-4be1a1f4-1d46-4766-a737-7eb8efcb65b4.png
+
+-  WebUI: Add support for displaying NaN and Infinity metric values.
+
+-  Support the `MMDetection <https://github.com/open-mmlab/mmdetection>`__ library in Model Hub to
+   easily train object detection models. MMDetection provides efficient implementations of popular
+   object detection methods like Mask R-CNN, Faster R-CNN, and DETR on par with Detectron2. In
+   addition, cutting-edge approaches from academia are regularly added to the library. See :ref:`the
+   documentation <model-hub-mmdetection>` for more information.
+
+-  Add the ability to use customizable master configuration templates in ``det deploy aws|gcp``.
+
+-  Images: Add an environment image for CPU-only TensorFlow 2.5 and 2.6.
+
+**Improvements**
+
+-  API: The aggregated historical resource allocation APIs
+   ``/api/v1/resources/allocation/aggregated`` and ``/allocation/aggregated`` now account for all
+   resources, not just those allocated to experiments.
+
+-  Images: Add CPU-only images for TF 2.5 and 2.6.
+
+-  Images: TF 2.5 and 2.6 images will no longer include PyTorch builds. For PyTorch 1.9, please use
+   the combined TF 2.4/PyTorch 1.9 image.
+
+-  Images: TF 2.4, 2.5, 2.6, and PyTorch 1.9 images will now use Python 3.8. The legacy TF
+   1.15/PyTorch 1.7 image will continue to use Python 3.7.
+
+**Changes**
+
+-  WebUI: Change the task list page to open new tabs when user clicks on task links.
+-  WebUI: The trial detail page will no longer show workload-based start time information, including
```

Comment:
non-blocking: I'm surprised the lack of space between two bullet points doesn't fail the linter!

Reply:
If all list items are under three lines, they are formatted with no blank lines between them. I have [my reasons](https://github.com/dzhu/rstfmt/commit/8676ef8d772b23d43981910385742153f7fb588f), but maybe this is [too surprising](https://github.com/dzhu/rstfmt/issues/13)? Might change at some point. :man_shrugging: